content/national-qualifications-framework/ncit/content/web-design-for-business/_index.md

```diff
@@ -20,7 +20,7 @@ ncit_standards:
 - 115374
 prerequisites:
   hard:
-  - projects/first-personal-website
+  - projects/web-development-website
```

Comment:
This should have the simple website as a prerequisite. Otherwise we are forcing the data scientists to do web dev on hard mode. We are trying to avoid that